Dockerfile

```diff
@@ -18,8 +17,15 @@ ENV NEXT_TELEMETRY_DISABLED 1
 
 RUN yarn build
 
-# If using npm comment out above and use below instead
-# RUN npm run build
+FROM golang:1.19 AS go
```

Comment:
1.20?

Reply:
We're building the tailscale go compiler, it theoretically shouldn't matter